web100/web100.go

```diff
@@ -100,28 +107,28 @@ func (w *Web100) logValues() (map[string]bigquery.Value, error) {
 	agent := C.web100_get_log_agent(log)
```

Comment:
Would prefer a name other than log, just to avoid confusion with logging, both here and w.log above.